src/styles/blocks/_button.scss

```
@@ -0,0 +1,23 @@
+.button{
+  display: flex;
+  width: 166px;
```

no needs to hardcode width
```suggestion
  width: 100%;
```